grafana-plugin/src/models/user/user.ts

```diff
@@ -304,4 +305,28 @@ export class UserStore {
     }
     return this.items[this.currentUserPk] as ApiSchemas['CurrentUser'];
   }
+
+  async addPersonalWebook(data: { webhook: string; context: object; }) {
+    await makeRequest(`/webhooks/set_personal_notification`, {
+      method: 'POST',
+      data,
+    });
+    await this.loadCurrentUser();
+    await this.updatePersonalWebhook();
+  }
+
+  async updatePersonalWebhook() {
+    const response = await makeRequest('/webhooks/current_personal_notification');
+
+    runInAction(() => {
+      this.personalWebhook = response;
+    });
+  }
```

Comment:
I also wasn't sure if I should fetch this in `loadCurrentUser` or only when the user visits the webhook tab in the settings.